docs/kb/kb0011-configuration-migration.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: cluedin
+title: Configuration Migration
+permalink: /kb/config-migrate
+nav_exclude: true
+tags: ["configuration", "migration"]
+is_kb: true
+---
+
+## On this page
+{: .no_toc .text-delta }
+1. TOC
+{:toc}
+
+This document covers the process for migrating CluedIn configuration from one environment to another environment. This is especially useful when developing your foundation on one environment and want to promote it to production.
+
+## Configuration Migration
+
+### Prerequisites
+- PowerShell Core
+- PowerShell 'CluedIn Product Toolkit' Module
+    Please reach out to CluedIn support if you would like this toolkit.
+- Service or Admin account to do the export and import
+- Access to the frontend whether it's public facing or private facing
+
+### Guide
+The product toolkit supports a number of methods of transferring configuration from one environment to another. The main two methods are running locally on your desktop, or alternatively using a pipeline platform such as Azure DevOps
+
+Below will guide you through both local and Azure DevOps, but you can technically use other methods.
+
+#### Locally
+
+This guide will be using the `import` and `export` scripts only. The toolkit also supports running the functions on their own, but this will not be covered as part of this documentation.
+
+To successfully run this, you will need the following information before proceeding:
+- Full url of your frontend
+- A backup path locally to store the json files exported
+- Id's of the data you want to backup
+
+{:.important}
+To get the Ids of specific parameters, you will need to use the frontend and navigate to the desired resource. In the url, you will then notice the id of a given object. This may be a guid or integer depending on what is viewed.
```

Comment:
```suggestion
To get the IDs of specific parameters, you will need to use the front end and navigate to the desired resource. In the URL, you will then notice the ID of a given object. This may be a GUID or integer depending on what is viewed.
```